server/routes/projects.js

```diff
@@ -52,4 +54,30 @@ router.route('/:projectId').get(async (req, res) => {
 	}
 });
 
+router
+	.route('/:projectId/comments')
+	.post(authenticateToken, async (req, res) => {
+		const { user } = req;
+		user._id = isValidObjectId(user._id);
+		user.username = isValidUsername(user.username);
```

Comment:
this should also be in the try block, this could throw 400 error as well

Reply:
Done